crates/sc-proof-of-time/src/time_keeper.rs

```diff
@@ -98,17 +98,19 @@ where
 
     /// Initializes the chain state from the consensus tip info.
     async fn initialize(&self) {
-        info!("time_keeper::initialize: waiting for initialization ...");
+        debug!("Waiting for initialization");
 
         // Wait for the first block import.
         let mut block_import = self.client.import_notification_stream();
         while let Some(incoming_block) = block_import.next().await {
             let pre_digest = match extract_pre_digest(&incoming_block.header) {
                 Ok(pre_digest) => pre_digest,
-                Err(err) => {
+                Err(error) => {
                     warn!(
-                        "time_keeper::initialize: failed to get pre_digest: {}/{:?}/{err:?}",
-                        incoming_block.hash, incoming_block.origin
+                        %error,
```

Comment:
The previous log statements were  quite concise and easier to read, please don't change these

Reply:
NVM, this looks good too (guess needs some getting used to)

Reply:
Structured logging is almost always better, you don't need to guess what the author concatenated in there and when JSON output is used it would be possible to do easy and efficient filtering of logs by these fields rather than parsing opaque strings.